tools/nnf-switch.sh

```diff
@@ -132,41 +132,43 @@ getChassis() {
 
 getPAXID() {
     local SWITCH_NAME=$1
+    bogusPAXID=9999
 
-    PAX_ID=$(switchtec fabric gfms-dump "$SWITCH_NAME" | grep "^PAX ID:" | awk '{print $3}')
-    ret=$?
-    if [ ! $ret ]; then
-        echo "Unable to retrieve PAX ID"
-        exit $ret
-    fi
-
-    if ! (( PAX_ID >= 0 && PAX_ID <= 1 )); then
-        echo "$PAX_ID not in range 0-1"
-        exit 1
+    PAX_ID=$(switchtec fabric gfms-dump "$SWITCH_NAME" | grep "^PAX ID:" | awk '{print $3}' || echo "$bogusPAXID")
+    if (( PAX_ID == "$bogusPAXID" ));
```

Comment:
The semicolon is needed only when you're going to put "then" on the same line with the "if".
But it's harmless as you have it.